.github/workflows/main.yml

```diff
@@ -10,4 +10,4 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Run xcodebuild with tests
-      run: xcodebuild test -scheme OmiseSDKTests -destination 'platform=iOS Simulator,name=iPhone 11,OS=13.6'
+      run: xcodebuild test -scheme OmiseSDKTests -destination 'platform=iOS Simulator,name=iPhone 11,OS=latest'
```

Comment:
This allows the tests to run and pass without constantly updating the version.

Reply:
Many thanks @danfowler 🙇